### ISSUE TITLE
Serial definitions do not look correct

### ISSUE THREAD

**Issue by adamvoss** [macchina/arduino-boards-sam#19]
I went to create a PR adding a `#define Serial SerialUSB` to better M2 compatibility with Due sketches.  In the process I noticed that Serial configuration looks to be unmodified from the Due version and would thus be incorrect because the M2 does not have all the same serial interfaces.

**Comment by TDoust** [comment]
Adam

I will look into this & get back with a correction

**Comment by TDoust** [comment]
Hi Adam

I have reviewed the serial definitions

```
/*
* UART/USART Interface
*/

// Serial
#define PINS_UART            98     // XBEE_RX, XBEE_TX

// Serial1
#define PINS_USART0          99     // LIN_KTX, LIN_RTX

// Serial2
#define PINS_USART1          100    // LIN_LTX, LIN_LRX

// Serial3
#define PINS_USART3          101    // UART3/TXD3, RXD3 26 pin connector
```

**Comment by adamvoss** [comment]
Resolved by #23
